src/sidebar/services/local-storage.ts

```diff
@@ -1,26 +1,24 @@
+type BasicStorage = Omit<Storage, 'length' | 'clear' | 'key'>;
```

Comment:
This is fine. We could also implement these properties/methods for `InMemoryStorage`.

Reply:
Ok, I have actually reverted this. It makes the code coverage drop, because there's no way to test those methods.

I could ignore them from code coverage, but since we are not using them, it probably makes no sense to maintain them.

Let's add them later if we actually find a use case.